acceptance/README.md

```diff
@@ -1,164 +1,439 @@
+```
 
-If you run a number of jobs with --preserve_hosts or vi ci:test_and_preserve_hosts, you may eventually generate a large number of stale vms.  They should be reaped automatically by qa infrastructure within a day or so, but you may also run:
+Beaker will install the following:
 
-    bundle exec rake ci:release_hosts
+```
+:install=>
+  ["git://github.delivery.puppetlabs.net/puppetlabs-facter.git#stable",
+   "git://github.delivery.puppetlabs.net/puppetlabs-hiera.git#stable",
+   "git://percival.corp.puppetlabs.net/puppetlabs-puppet.git#abcd"],
+```
 
-to clean them up sooner and free resources.
+This corresponds to installing facter and hiera stable from our internal mirror, while installing puppet SHA abcd from a git daemon on my local machine percival.  See below for details on setting up a local git daemon.
 
-There also may be scenarios where you want to specify the host(s) to release. E.g. you may want to release a subset of the hosts you've created. Or, if a test run terminates early, ci:release_hosts may not be able to derive the name of the vm to delete. In such cases you can specify host(s) to be deleted using the HOST_NAMES environment variable. E.g.
 
-    HOST_NAMES=lvwwr9tdplg351u bundle exec rake ci:release_hosts
-    HOST_NAMES=lvwwr9tdplg351u,ylrqjh5l6xvym4t bundle exec rake ci:release_hosts
+### Authentication
 
-### Syncing local development code with VMPooler Windows machines
+Normally the ci tasks are called from a prepared Jenkins job.
 
-If you want to copy local code to a Windows machine that is running in beaker, you can use the `ci:sync:windows` rake task:
+If you are running this on your laptop, you will need this ssh private key in order for beaker to be able to log into the vms created from the hosts file:
 
-    WIN_MACHINE=edqa6se2xxo5r21 rake ci:sync:windows
+https://github.com/puppetlabs/puppetlabs-modules/blob/production/secure/jenkins/id_rsa-acceptance
+https://github.com/puppetlabs/puppetlabs-modules/blob/production/secure/jenkins/id_rsa-acceptance.pub
 
-Optional environment variables for this are:
+Please note in acceptance/Rakefile where the ssh key is defaulted to. It may be looking in ~/.ssh/id_rsa-acceptance, but it may want to look in the working directory (e.g. puppet/acceptance).
 
-    LIB_DIR (defaults to `type`) - Path in `lib/puppet` to sync with the Windows machine.
+You will also need QA credentials to vsphere in a ~/.fog file.  These credentials can be found on any of the Jenkins coordinator hosts. You may want to check periodically to ensure that the credentials you have are still valid as they may change periodically.
 
 
-Running Tests on Vagrant Boxen
-------------------------------
+### Running Tests on Vagrant Boxen
 
 This guide assumes that you have an acceptable Ruby (i.e. 1.9+) installed along with the bundler gem, that you have the puppet repo checked out locally somewhere, and that the name of the checkout folder is `puppet`.
```

Comment:
We might want to update the version of what's considered "acceptable Ruby"? And test through this section to make sure all the steps work and behave the same as they're documented here?

Reply:
Let's just 🔥 the old cruft.